packages/esm-patient-chart-app/src/visit/hooks/useServiceQueue.tsx

```diff
@@ -75,3 +84,15 @@ export function saveQueueEntry(
     body: payload,
   });
 }
+
+export function useQueueLocations() {
+  const apiUrl = `${fhirBaseUrl}/Location?_summary=data&_tag=queue location`;
+  const { data, error } = useSWR<{ data: FHIRResponse }>(apiUrl, openmrsFetch);
```

Comment:
`const { data, error, isLoading } = useSWR<{ data: FHIRResponse }>(apiUrl, openmrsFetch);`

With recent updates to swr, we have the `isLoading` being retrieve directly from the hook